sphinx_gallery/tests/test_full.py

```diff
@@ -66,5 +66,15 @@ def test_embed_links(sphinx_app):
     assert 'scipy.signal.firwin.html' in lines
     assert '#module-numpy' in lines
     assert 'numpy.arange.html' in lines
-    # assert '#module-matplotlib.pyplot' in lines
-    # assert 'pyplot.html' in lines
+    assert '#module-matplotlib.pyplot' in lines
+    assert 'pyplot.html' in lines
+
+
+def test_backreferences(sphinx_app):
+    """Test backreferences."""
+    out_dir = sphinx_app.outdir
+    mod_file = op.join(out_dir, 'gen_modules', 'sphinx_gallery.sorting.html')
+    with codecs.open(mod_file, 'r', 'utf-8') as fid:
+        lines = fid.read()
+    assert 'FileNameSortKey' in lines
```

Comment:
Very nice test!

Maybe add a few comments before each assert to explain a bit what they are checking? Proposal (improvements more than welcome!):
* first assert: "check that FileNameSortKey class appears in the reference API of the sphinx_gallery.sorting module"
* second assert: "check that the backreferences work when 
  
Bonus points if you can add a test that the "normal" (i.e. in Python code) backreferences works? (maybe there is one already I haven't checked ...)